src/xrEngine/xr_ioc_cmd.cpp

```diff
@@ -442,11 +442,7 @@ class CCC_VidMonitor : public CCC_Token
         bEmptyArgsHandled = false;
     }
 
-    void Execute(pcstr args) override
-    {
-        CCC_Token::Execute(args);
-        FillVidModesToken(Vid_SelectedMonitor);
-    }
```

Comment:
This is intended, don't remove this please.

Reply:
For what?

Reply:
When you are changing monitor you need to update vid mode token. That's why.